tools/seahorn/seahorn.cpp

```diff
@@ -143,13 +142,19 @@ static llvm::cl::opt<bool>
             "Use BMC. Currently restricted to intra-procedural analysis"),
         llvm::cl::init(false));
 
-static llvm::cl::opt<seahorn::bmc_engine_t>
+// Available BMC engines
+typedef enum {
```

Comment:
I realy like new class enums of C++11. Do you mind switching to those. Otherwsie enum values pollute the global namespace.